src/__tests__/ParseObject-test.js

```diff
@@ -1219,6 +1219,26 @@ describe('ParseObject', () => {
     });
   });
 
+  it('accepts context on save', async () => {
```

Comment:
Looks good, can you add an integration test?

Reply:
I thought about this, but this PR requires a PR in parse server and vice versa. How would I set up an integration test?

Reply:
Btw, the PR in parse server has a test case which tests the integration.

Reply:
Oh right I didn't see that.

Reply:
No worries, let me know if anything else to take a look at...